tools/bouncer_worker/src/lib/config.js

```diff
@@ -57,13 +57,13 @@ const bouncer = object({
 });
 
 const elastic = object({
-	cloud: envars,
-	auth: envars,
+	cloud: object({id: string()}).default(undefined),
```

Comment:
I don't think you need the default(undefined) here, unless there's situation where someone would put in an elastic object and the application should still be functional without the cloud/auth definition.

In fact I think most of these fields should be required?

Things like maxRetries/reload_connections etc, unless the code checks for if they're empty and otherwise don't do anything, we want to impose a default?

Reply:
Can we check through all the fields and determined if thye're required, optional, or should have default value?

Reply:
I don't think the majority of these fields actually mean anything to Elasticsearch 7.17 :rofl: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/7.17/basic-config.html

Reply:
@carmenfan I've removed those that aren't supported. Do we care about adding

Reply:
no, if we don't know about it then we probably don't pass it along i'm guessing?